code/modules/clothing/head/armor.dm

```diff
@@ -538,3 +538,112 @@
 	name = "gray tanker helmet"
 	icon_state = "tanker_helmet_gray"
 
+/obj/item/clothing/head/armor/faceshield/paramedic
+	name = "Moebius paramedic helmet"
+	desc = "Seven minutes or a refund."
+	icon_state = "trauma_team"
+	item_state = "trauma_team"
+	flags_inv = HIDEEARS|BLOCKHAIR
+	item_flags = BLOCK_GAS_SMOKE_EFFECT|AIRTIGHT
+	matter = list(
+		MATERIAL_PLASTEEL = 10,
+		MATERIAL_GLASS = 5,
+		MATERIAL_PLASTIC = 5,
+		MATERIAL_PLATINUM = 2
+		)
+	armor_up = list(
+		melee = 25,
+		bullet = 25,
+		energy = 20,
+		bomb = 10,
+		bio = 100,
+		rad = 50
+		)
+	armor_down = list(
+		melee = 35,
+		bullet = 35,
+		energy = 30,
+		bomb = 15,
+		bio = 100,
+		rad = 50)
+	up = TRUE
+	spawn_blacklisted = TRUE
+	var/speaker_enabled = TRUE
+	var/scan_scheduled = FALSE
+	var/scan_interval = 15 SECONDS
+	var/repeat_report_after = 60 SECONDS
+	var/list/crewmembers_recently_reported = list()
+
+
+/obj/item/clothing/head/armor/faceshield/paramedic/equipped(mob/M)
+	. = ..()
+	schedule_scan()
+
+
+/obj/item/clothing/head/armor/faceshield/paramedic/proc/schedule_scan()
+	if(scan_scheduled)
+		return
+	
+	if(!speaker_enabled)
+		return
+
+	scan_scheduled = TRUE
+	spawn(scan_interval)
+		if(QDELETED(src))
+			return
+		scan_scheduled = FALSE
+		report_health_alerts()
+
+
+/obj/item/clothing/head/armor/faceshield/paramedic/proc/schedule_memory_cleanup(entry)
+	spawn(repeat_report_after)
+		if(QDELETED(src))
+			return
+		crewmembers_recently_reported.Remove(entry)
+
+
+/obj/item/clothing/head/armor/faceshield/paramedic/proc/report_health_alerts()
+	if(!speaker_enabled)
+		return
+
+	if(!loc || !istype(loc, /mob/living/carbon/human))
+		return
+	
+	var/mob/living/carbon/human/user = loc
+
+	var/list/crewmembers = list()
+	var/list/z_levels_to_scan = list(1, 2, 3, 4, 5)
```

Comment:
you should iterate through all z-levels marked with station Z_level round start , not hard code it.

Reply:
Helmet reacted to mannequin characters, so I set Z-levels to scan this way.